boilerplate/src/shared/styles/mixins/typography.css

```diff
@@ -0,0 +1,110 @@
+@import "../custom-medias";
+
+/* ==========================================================================
+  Fonts
+  ========================================================================== */
+
+@define-mixin font-montserrat $fontWeight: 400 {
+    font-family: "Montserrat", sans-serif;
+    font-weight: $(fontWeight);
+}
+
+/* ==========================================================================
+  All typography variants
+
+  $unit - Either em or rem
+  ========================================================================== */
+
+@define-mixin typography-numbers $unit {
+    @mixin font-montserrat 700;
+    font-size: 6$(unit);
+    line-height: calc(5.8em / 6);
+
+    @media (--layout-lte-tablet) {
+        font-size: 4.8$(unit);
+        line-height: calc(5.8em / 4.8);
+    }
+}
+
+@define-mixin typography-h1 $unit {
+    @mixin font-montserrat 600;
```

Comment:
Normally I think is better separate mixins from the rest of the rules.
What do you think?

```
@define-mixin typography-h1 $unit {
     @mixin font-montserrat 600;

     font-size: 4.2$(unit);
     line-height: calc(5.8 / 4.2);
}